src/ConfigSetup.cpp

```diff
@@ -224,9 +224,19 @@ void ConfigSetup::Init(const char *fileName, MultiSim const*const& multisim)
     } else if(CheckString(line[0], "FirstStep")) {
       in.restart.step = stringtoi(line[1]);
     } else if(CheckString(line[0], "PRNG")) {
-      in.prng.kind = line[1];
-      if("RANDOM" == line[1])
+      if("RANDOM" == line[1]){
         printf("%-40s %-s \n", "Info: Random seed", "Active");
+        in.prng.kind = line[1];
+      } else if ("INTSEED" == line[1]){ 
+        printf("%-40s %-s \n", "Info: Integer seed", "Active");
+        in.prng.kind = line[1];
+      } else if ("RESTART" == line[1]) {
+        printf("%-40s %-s \n", "Info: Restart seed", "Active");
+        in.prng.kind = line[1];
```

Comment:
What's the point of this?

Reply:
Removed.  That was leftover from when I was trying to fix PRNG Kind Restart.